BlueSky_pygame.py

```diff
@@ -38,4 +41,4 @@ def MainLoop():
 # Run mainloop if BlueSky_pygame is called directly
 
 if __name__ == '__main__':
-    MainLoop()
+    mainloop()
```

Comment:
Prolly a lot of these case-y problems. I like the idea of gently cleaning them out as we go, but may get pushback. I'd use `main_loop` for equivalence with their CamelCasing (camel_casing).